index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <title>PORTFOLIO</title>
+</head>
+<body>
+    <header>
+        <ul class="toolbar">
+            <li class="welcome"><a href="#">Welcome</a></li>
+            <li class="menu"><a href="#">☰</a></li>
+
+        </ul>
+    </header>
+    <div class="headline">
+        <h1>I’m Amine Glade to see you!</h1>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+        <span><a>LET'S CONNECT <br> </a></span>
+        <img src="/social-media-icons.png" alt="Social Media icons">
+    </div>
```

Comment:
- [ ]  Please remove the underline from the left side logo ( "**Welcome**" ) and then try to match the color with the Figma design. :+1: 

- [ ] The hamburger menu on the right side is not matched with the Figma design. Please fix it. :+1: 

- [ ] The headline content is not matched with the Figma design. Please work on the font size, color, and proper margin and padding to match the Figma design. :+1: 

- [ ] **"Let's connect"** button color and margin are not matched with the Figma design. You can use `color: #7F8CFF;`  and also try to add some top and bottom margin for it.

- [ ] Your design is not matched with the Figma design properly. You can see the differences in the following screenshots. Kindly work on it. You can use [CSS Media Queries](https://www.w3schools.com/css/css3_mediaqueries.asp) for your mobile version to make it responsive well.

 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![m1](https://user-images.githubusercontent.com/70483835/164578708-dbbb8967-4d1a-4202-a3db-0cfb96ae450f.png) | ![m2](https://user-images.githubusercontent.com/70483835/164579625-17bb8af1-4719-42b7-bc8c-20fc0f7c1616.png)

Reply:
> * [ ]   Please remove the underline from the left side logo ( "**Welcome**" ) and then try to match the color with the Figma design. +1
> * [ ]  The hamburger menu on the right side is not matched with the Figma design. Please fix it. +1
> * [ ]  The headline content is not matched with the Figma design. Please work on the font size, color, and proper margin and padding to match the Figma design. +1
> * [ ]  **"Let's connect"** button color and margin are not matched with the Figma design. You can use `color: #7F8CFF;`  and also try to add some top and bottom margin for it.
> * [ ]  Your design is not matched with the Figma design properly. You can see the differences in the following screenshots. Kindly work on it. You can use [CSS Media Queries](https://www.w3schools.com/css/css3_mediaqueries.asp) for your mobile version to make it responsive well.
> 
> Your design	Figma Design
> ![m1](https://user-images.githubusercontent.com/70483835/164578708-dbbb8967-4d1a-4202-a3db-0cfb96ae450f.png)	![m2](https://user-images.githubusercontent.com/70483835/164579625-17bb8af1-4719-42b7-bc8c-20fc0f7c1616.png)

- [ ] Please work on the above-suggested changes.